ntbs-service/Pages/Alerts/TransferRequestAction.cshtml.cs

```diff
@@ -128,7 +128,7 @@ public async Task AcceptTransferAndDismissAlertAsync()
             var transferInEvent = new TreatmentEvent
             {
                 NotificationId = NotificationId,
-                EventDate = TransferRequest.TransferDate.AddSeconds(1),
+                EventDate = TransferRequest.TransferDate.Value.AddSeconds(1),

```

Comment:
If the new transfer date is the same as a previous transfer, then I think we need to make sure that the backdated transfer happens after the existing transfer